r/R/embedding.R

```diff
@@ -83,3 +61,67 @@ runEmbedding <- function(req, data) {
   res <- purrr::map2(df_embedding[[1]], df_embedding[[2]], map2_fun)
   return(res)
 }
+
+
+# getEmbedding
+# Return embedding calculated for the seurat object.
+# data is the seurat object.
+# config is the embedding work request config.
+# type is the embedding type.
+# reduction_type is the type of reduction that is used, e.g. PCA.
+# num_pcs is the number of principal components.
+#
+# Req$body has:
```

Comment:
Is this empty because it still has to be defined?

Reply:
No, we're not using $req anymore. removed.